content/webapp/pages/search/index.tsx

```diff
@@ -161,23 +166,44 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
           <SearchNoResults query={queryString} />
         </Container>
       ) : (
-        <Container style={{ display: 'flex' }}>
+        <Container
+          style={{
+            display: 'grid',
+            gridTemplateColumns: '3fr 2fr',
+            gap: '160px',
+          }}
+        >
           <BasicSection>
-            <Container>
-              <SectionTitle sectionName="Content results" />
-              <p>{contentResults?.totalResults || 0} results</p>
-              <div
-                style={{
-                  fontSize: '12px',
-                  maxWidth: '780px',
-                  overflow: 'scroll',
-                }}
+            <p className={font('intr', 5)}>
+              {contentQueryFailed ? (
+                <>
+                  There was a problem fetching some search results. Please try
```

Comment:
This would need an editor to write it for real

Reply:
Fairs, could create a new ticket for error messages or add it to the no results scenarios as those will be worked on next sprint?

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/11463#issuecomment-2619658736